src/shared/lib/CivicUtils.ts

```diff
@@ -30,6 +30,8 @@ export function getCivicGenes(geneSymbols: Array<string>): Promise<ICivicGene> {
     let promises: Array<Promise<Array<ICivicGeneData>>> = [];
     let ids = '';
     geneSymbols.forEach(function(geneSymbol: string) {
+        //Encode "/" characters
+        geneSymbol = geneSymbol.replace(/[\/]/g,'%2F');
```

Comment:
minor quibble, but i don't think you need the [] brackets here.  you would use them if you were specifying a class of acceptable chars.   but here there's ony one, so /\// will work.